### ISSUE TITLE
Code snippets in Preview README.md are hard to read in Light High Contrast theme

### ISSUE THREAD

**Issue by v-amyang** [microsoft/vscode-azurefunctions#5071]
**OS**: Win 11 x64
**Build Version**: [1.21.3](https://github.com/microsoft/vscode-azurefunctions/actions/runs/26667473388)

**Repro Steps:**
1. Start creating a project using Template Gallery
2. Open Template Details for any template and proceed with project creation
3. After creation, Preview README.md is opened automatically
4. Switch VS Code theme to "Light High Contrast"
5. Check the readability of code snippets in the README.md

**Expect:**
Code snippets in the Preview README should be clearly readable with sufficient contrast in Light High Contrast theme

**Actual:**
Code snippets in the Preview README are difficult to read in Light High Contrast theme
<img width="901" height="752" alt="Image" src="https://github.com/user-attachments/assets/6c5f573f-715d-460b-b20c-0c63ee6e0a30" />

**Comment by gavin-aguiar** [comment]
This is not an issue with the template gallery. The readme is from the Azure samples repo.  Even if the user clones the code and opens it with this theme, it will look the same.
